ts/components/messages/paginated/MessageList/Item.tsx

```diff
@@ -2,8 +2,10 @@ import React from "react";
 import { StyleSheet } from "react-native";
 import { Badge, Text, View } from "native-base";
 
+import LegalMessage from "../../../../../img/messages/legal-message.svg";
 import { ServicePublic } from "../../../../../definitions/backend/ServicePublic";
 import { MessageCategory } from "../../../../../definitions/backend/MessageCategory";
+import { TagEnum } from "../../../../../definitions/backend/MessageCategoryBase";
```

Comment:
With a small device and a very long organization name we could have an overflow:

<img width="383" alt="Schermata 2021-12-23 alle 15 09 02" src="https://user-images.githubusercontent.com/26501317/147251631-7c43985c-7774-415a-a7de-ffa30a8f1180.png">

my suggestion is to change the `titleRow` style adding `flex: 1` and create a style for the component:
```typescript
StyleSheet.create({
...
organizationName:{
   flex:1,
   paddingRight: 16 
},
...
})

...

<H5 style={styles.organizationName} numberOfLines={1}>
          {organizationName}
        </H5>
```
with the addition of `numberOfLines={1}`
<img width="389" alt="Schermata 2021-12-23 alle 15 20 20" src="https://user-images.githubusercontent.com/26501317/147252952-0543fa2f-f34c-4217-b551-9f9ddc93bcad.png">

Reply:
Good catch! Fixed https://github.com/pagopa/io-app/pull/3599/commits/013f02522f9a7bce8f44ee89f4a5eca50e1ae8e5